tests/vmi_configuration_test.go

```diff
@@ -2941,6 +2941,12 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		})
 	})
 	Context("virt-launcher processes memory usage", func() {
+		doesntExceedMemoryUsage := func(processRss *map[string]resource.Quantity, process string, memoryLimit resource.Quantity) {
+			actual := (*processRss)[process]
+			Expect((&actual).Cmp(memoryLimit)).To(Equal(-1),
```

Comment:
Can you use `ExpectWithOffSet` for a better pointer to which line triggers this?
Also, I think you don't need to pass the map as a pointer but I let you decide.

Reply:
Could you please explain `ExpectWithOffSet` to me? I've always been confused by it...
As far as the map goes, if I don't pass it as a pointer, won't it get duplicated in memory?
Thanks!

Reply:
Of course with pleasure. 
Ginkgo will provide you with a line where the assertion failed. As written now it would say it failed on line 2946.
By using the `ExcepWithOffSet` with 1 it will go one call stack back. Meaning it will show the line where this function got called.

As for the map, I think only the header gets copied not the data([see](https://github.com/golang/go/blob/master/src/runtime/map.go#L116)).